app/models/blog.rb

```diff
@@ -10,7 +10,7 @@ class Blog < ApplicationRecord
   scope :published, -> { where('secret = FALSE') }
 
   scope :search, lambda { |term|
-    where("title LIKE '%#{term}%' OR content LIKE '%#{term}%'")
+    where('title LIKE? OR content LIKE?', "%#{term}%", "%#{term}%")
```

Comment:
```suggestion
    where('title LIKE ? OR content LIKE ?', "%#{term}%", "%#{term}%")
```

スペースを空けた方がいいですね。

あと、

```suggestion
    where('title LIKE :hoge OR content LIKE :hoge', "%#{term}%", "%#{term}%")
```

みたいに名前付きにすると引数を減らし、DRYにできます

Reply:
そんな書き方が...😳ありがとうございます！